docs/docs/cmd/spo/web/web-roleassignment-remove.mdx

```diff
@@ -57,6 +63,12 @@ Remove roleassignment from web based on principal Id without prompting for confi
 m365 spo web roleassignment remove --webUrl "https://contoso.sharepoint.com/sites/contoso-sales"  --principalId 2 --force
 ```
 
+Remove roleassignment from web based on Entra Group Id
```

Comment:
lets add a remark that this is force delete as it someone just do a copy/past and execute than this may have some not pleasent implications 

```suggestion
Remove roleassignment from web based on Entra Group Id and don't prompt for confirmation
```